### PR TITLE
heading 7 complete

### PR DESCRIPTION
**Pull request by musaab03**

### CODE REVIEW

File: models/reviews.js [hunk 1]
```diff
@@ -12,6 +15,14 @@ exports.selectReviewById = (id) => {
     });
 };
 
+exports.selectComments = (id) => {
```
Comment (musaab03) [inline models/reviews.js:18]: ignore lines 18-24 i forgot to remove this bit

File: __tests__/app.test.js [hunk 2]
```diff
@@ -55,11 +55,29 @@ describe("GET", () => {
...
+        });
+    });
 
+    test("200: for returning review from id with no comments", () => {
+      return request(app)
+        .get("/api/reviews/1")
+        .expect(200)
+        .then((response) => {
+          const { body } = response;
+          console.log("body.review :>> ", body);
```
Comment (pkaodev) [inline __tests__/app.test.js:80]: remove console.log

File: __tests__/app.test.js [hunk 3]
```diff
@@ -68,11 +86,12 @@ describe("GET", () => {
           expect(typeof body.review.review_img_url).toBe("string");
           expect(typeof body.review.created_at).toBe("string");
           expect(typeof body.review.votes).toBe("number");
```
Comment (pkaodev) [inline __tests__/app.test.js:88]: these should be the actual values from review 1 as well

**Review by pkaodev — APPROVED** [review]
just a few changes in your tests

**Pull request closed without merging**